quickstart/app/helper/function.php

```diff
@@ -0,0 +1,20 @@
+<?php
+function GetCategory($mang,$parent,$shift,$active)
+{
+    foreach($mang as $row)
+    {
+        if($row->parent==$parent)
+        {
+            if($row->id==$active)
+            {
+                echo "<option selected value='$row->id'>".$shift.$row->name."</option>";
+            }
+            else {
+                echo "<option value='$row->id'>".$shift.$row->name."</option>";
+            }
+            
+            GetCategory($mang,$row->id,$shift.'---|',$active);
```

Comment:
tên hàm viết in thường chữ cái đầu, thêm dấu cách vào sau dấu phẩy. cách ra giữa dấu "." nối 2 string